src/components/Footer/index.js

```diff
@@ -0,0 +1,63 @@
+import React, { PureComponent } from 'react'
+import cn from 'classnames'
+
+import logo from '../../assets/logo.svg'
+import sslBadge from '../../assets/ssl-badge.svg'
+
+export default class Footer extends PureComponent {
+  infoBlock = React.createRef()
+
+  render () {
+    return (
+      <div id="mc-footer">
```

Comment:
should we use `id`s for the html elements? I think is a good practice to use classes instead of `id`s.. But I'm not sure if there is a good reason to use `id`s here

Reply:
I implemented the header and footers with IDs originally because there was only ever supposed to be one and we'd have issues if there were more than one on the page.  If we think we could possibly use two footers in the future, I can update these to classes no problem, but I think for something as unique as a header and footer, specifically "the" header and footer, IDs are correct semantically

Reply:
Makes sense